### ISSUE TITLE
Jy Conversions

### ISSUE THREAD

**Issue by graceparkk** [hover2pi/sedkit#75]
Hi! I've downloaded the new SEDkit and used Jy as the units for flux; however, I found that it has wrong conversions. With u.Jy, I get this plot. 
![bokeh_plot (10)](https://user-images.githubusercontent.com/108310946/184424581-25702be8-f9a2-44e0-bbdd-8020722ede24.png)
 
Without using u.Jy and using this conversion, the SED plot seems to be right. 
```
unit_flux1 = flux1*u.Jy.to(u.erg/u.s/u.cm**2/u.AA, equivalencies=u.spectral_density(wl1*u.micron))
unit_unc1 = eflux1*u.Jy.to(u.erg/u.s/u.cm**2/u.AA, equivalencies=u.spectral_density(wl1*u.micron))
sed.add_spectrum(spectrum=[wl1*u.micron, unit_flux1*u.erg/u.s/u.cm**2/u.AA, unit_unc1*u.erg/u.s/u.cm**2/u.AA])
```
![bokeh_plot (11)](https://user-images.githubusercontent.com/108310946/184424873-090a0fbe-d0b1-4848-a3c4-c2b0999892f2.png)

I think that it's the problem with converting Jy wrong.

**Comment by hover2pi** [comment]
Thanks for the issue @graceparkk ! I think I know what the problem is. I'll fix it ASAP.

**Comment by hover2pi** [comment]
It's hard to debug this without the data. Your figures aren't comparing apples to apples here. The first plot has no WISE photometry and shows the potential problem for sure. However, your second plot does have WISE mags so the code is normalizing the spectrum to that. Can you make the second plot again but don't add the WISE photometry?

I'm going to incorporate `astropy.units` to do the conversion and then write some explicit verification tests to make sure the conversion is right in the next release. Thanks!

**Comment by graceparkk** [comment]
![bokeh_plot (2)](https://user-images.githubusercontent.com/108310946/185679608-e3e4cb53-f00c-45c2-8f88-8774d2bdaa7f.png)
![bokeh_plot (3)](https://user-images.githubusercontent.com/108310946/185679626-cbbc1139-3e15-46c7-a457-c162e8472fd5.png)
I've added the WISE photometries for both plots, but I still get a difference in the SED plots. The upper one is using u.Jy and the lower one is the one I've manually converted using the code I mentioned above. Before the new release, I will continue using the manual conversions. 
Thank you for your help!
